presto-docs/src/main/sphinx/release/release-0.257.rst

```diff
@@ -0,0 +1,29 @@
+=============
+Release 0.257
+=============
+
+**Details**
+===========
+
+General Changes
+_______________
+* Fix scheduling issue in join spill to improve memory used by unspilled data.
+* Fix spilled query exceeding local memory limit due to incorrect memory size calculation when spilled data is read by SingleStreamSpiller.
```

Comment:
Fix query exceeding local memory limit when unspilling data due to incorrect memory size calculation.

Reply:
done